### DISCUSSION TITLE
Something missing on the pip downloaded pycord...

### DISCUSSION THREAD

**Discussion by larpedd** [Pycord-Development/pycord#274] [Q&A] [answered]
Hi!, I just installing py-cord winch like normal, but when i make a bot variable (Just writing like the bot set-up on the examples). that didn't worked, in the console says 
```python
Traceback (most recent call last):
  File "C:\Users\MY\Desktop\h\main.py", line 3, in <module>
    bot = discord.Bot()
AttributeError: module 'discord' has no attribute 'Bot'
```
Then i open the discord site-pacakages, and i found something missing,
I matched between the github and the discord site packages.

![image](https://user-images.githubusercontent.com/53142287/137591623-54186476-9790-40dc-bbf2-cd98cf199e91.png)

I was like 'Why it just the ''backoff.py"?', i open the ext/commands/ and i found the bot.py sitting in there. Can you tell me why?.

Thanks.

**Comment by izxxr** [comment, marked as answer]
You have to install using Git to use the features from master branch.
```sh
pip install git+https://github.com/Pycord-development/pycord.git
```
Note that you need to have Git installed.

**Reply by larpedd** [reply]
Thanks, It worked!
